crates/core-engine/src/data/mod.rs

```diff
@@ -0,0 +1,6 @@
+pub mod enemy_config;
+pub mod player_config;
+pub mod enemy_configs;
+
+pub const player_config_str: &str = include_str!("./player_config.ron");
+pub const enemy_configs_str: &str = include_str!("./enemy_configs.ron");
```

Comment:
just adds it as a constant during compile time. possibly not the best, but also, really not bad.

Reply:
I don't know how `.ron` format flew under my radar but that's super convenient